CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## [1.1.0]
+- Update dependencies
+- Add linting
+- Migrate example to null-safety
+- Deprecate [onLinkTap] in favor of [beforeLaunch]
+- Update known URI schemes and TLDs
```

Comment:
did you do this with the scripts soru provided?

Reply:
Nope.

Reply:
why not? those are generating from official sources?

Reply:
Which script do you actually mean?

Reply:
Ah, you only mean the schemes and TLDs? Of course! How else?